vizro-core/src/vizro/models/_models_utils.py

```diff
@@ -20,3 +20,25 @@ def _validate_min_length(cls, field):
     if not field:
         raise ValueError("Ensure this value has at least 1 item.")
     return field
+
+
+# Based on how Github generates anchor links - see:
+# https://stackoverflow.com/questions/72536973/how-are-github-markdown-anchor-links-constructed.
+def _clean_path(path: str, allowed_characters: str, enable_url: bool = True) -> str:
```

Comment:
This function had to be refactored for the sake of creating valid CSS IDs e.g. a valid CSS ID cannot start with` / `

Reply:
Just for my own tracking: We've decided to **not clean** the path, but instead add to the docs that the user has to provide a valid `id`. I'll add to the docs what we consider as `valid`:
- Use lowercase letters and hyphens
- Avoid using special characters or spaces
- Provide a unique `id`